src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy

```diff
@@ -126,21 +120,17 @@ class ModuleExtension
         modProperties.setProperty("BuildPath", project.buildDir.getAbsolutePath())
         modProperties.setProperty("SourcePath", project.projectDir.getAbsolutePath())
         modProperties.setProperty("ResourcePath", "") // TODO  _project.getResources().... ???
-        modProperties.setProperty("LabkeyVersion", (String) project.getProperty("labkeyVersion"))
-        boolean isExternalModule = project.projectDir.getAbsolutePath().contains("externalModules")
-        if (modProperties.getProperty("ConsolidateScripts") == null)
+        modProperties.setProperty("ReleaseVersion", (String) project.getProperty("labkeyVersion"))
+        if (modProperties.getProperty("ManageVersion") == null)
         {
-            if (isExternalModule)
-                modProperties.setProperty("ConsolidateScripts", "false")
-            else
-                modProperties.setProperty("ConsolidateScripts", "true")
+            modProperties.setProperty("ManageVersion", "true")
         }
-        if (modProperties.getProperty("ManageVersion") == null)
+        if (modProperties.getProperty("SchemaVersion") == null)
         {
-            if (isExternalModule)
-                modProperties.setProperty("ManageVersion", "false")
+            if (modProperties.getProperty("Version") == null)
+                modProperties.setProperty("SchemaVersion", "")  // Spring binds this as setSchemaVersion(null), which is what we want
             else
-                modProperties.setProperty("ManageVersion", "true")
+                modProperties.setProperty("SchemaVersion", modProperties.getProperty("Version"))  // For backward compatibility with old modules TODO: Remove
```

Comment:
Do we never use want to use the SchemaVersion value? (I assume that's better handled in the code itself, but a comment to that effect would be handy here.)

Reply:
We do use it... if non-null, it gets copied into the template. This is the SchemaVersion == null (not specified in module.properties) case. Maybe I'm not understanding what you're asking for...

Reply:
Ah, right.  Wasn't seeing the else part.